### DISCUSSION TITLE
You must also enable the Generate Type Declarations options on the Customer class for the attributes to be compatible with the Interface

### DISCUSSION THREAD

**Discussion by baladpi** [pimcore/pimcore#17268] [Q&A / Support]
IN https://pimcore.com/docs/platform/Customer_Management_Framework/Installation/  - in the installation process, the line mentioned as  "You must also enable the Generate Type Declarations options on the Customer class for the attributes to be compatible with the Interface" but Generate Type Declarations options is not available.

**Comment by jdreesen** [comment]
It was optional in Pimcore 10 but is always enabled in Pimcore 11. The docs are outdated.

**Reply by jdreesen** [reply]
I created a PR to update the docs: https://github.com/pimcore/customer-data-framework/pull/538
